### PR TITLE
Port latest SDK options - React Server

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
update sample with latest sdk options

- disableGrammar
- disableTranslation
- disableLanguageDetection
- parent

### CODE REVIEW

File: js/samples/react-sample-server/client/src/AppOptions.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    options.disableGrammar = sampleId === 'DisableGrammar';
+    options.disableTranslation = sampleId === 'DisableTranslation';
+    options.parent = sampleId === 'Parent' && document.getElementById("checkboxParent").checked ? document.getElementById('parentDiv') : null;
+
+    try {
+      await launchAsync(token, subdomain, data, options)
+    }
+    catch (error) {
+      alert("Error in launching the Immersive Reader. Check the console.");
+      console.log(error);
```
Comment (rwallerms) [inline js/samples/react-sample-server/client/src/AppOptions.js:78]: log error first

File: js/samples/react-sample-server/client/src/AppOptions.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    }
+  }
+
+  const handleLanguage = (e) => setLanguage(e.target.value);
+
+  useEffect(() => {
```
Comment (rwallerms) [inline js/samples/react-sample-server/client/src/AppOptions.js:84]: add comment how this works

**Review by rwallerms — CHANGES_REQUESTED** [review]

File: js/samples/react-sample-server/client/src/AppOptions.js (hunk 1 above)
Comment (cristobal-buenrostro) [reply js/samples/react-sample-server/client/src/AppOptions.js:78]: done, also in the other page Apps.js

File: js/samples/react-sample-server/client/src/AppOptions.js (hunk 2 above)
Comment (cristobal-buenrostro) [reply js/samples/react-sample-server/client/src/AppOptions.js:84]: added same comment as in App.js

**Review by rwallerms — APPROVED** [review]

**Pull request merged**